src/StatusQ/Core/StatusIcon.qml

```diff
@@ -3,7 +3,7 @@ import QtGraphicalEffects 1.13
 
 Image {
     property string icon: ""
-    property color color
+    property string color: ""
```

Comment:
It seems `property color color` does get a default value that is not `undefined` or `""` even when turned into a string using `toString()`

We need an uninitialized color so changing this to `string` helps.

@borismelnik lemme know if there's a better way to check for `!!color` if `color` is of type `color`

Reply:
i think we can use "transparent" color by default and check it with `Qt.colorEqual(color, "transaprent")` bc image cant e invisible)))no ways in docs about it :(

Reply:
That worked perfectly - thanks!!